node/src/main/resources/defaults.conf

```diff
@@ -287,6 +287,12 @@ casper {
     # How many confirmations are necessary to use multi-sig vault.
     # The value should be less or equal the number of PoS multi-sig public keys.
     pos-multi-sig-quorum = 2
+
+    # Public key (default value) for genesis block
```

Comment:
Instead of `Public key (default value) for genesis block` please change to:
`Public key for transfers from the PoS vault`

---

We don't want default value to be specified because user who starting the network will not be able to use it. We need add warning message to the log is this value is empty when genesis block is created.
@zsluedem Please help Max to do that.